qfdmo/models/acteur.py

```diff
@@ -526,9 +511,26 @@ def is_parent(self):
     def save(self, *args, **kwargs):
         # OPTIMIZE: if we need to validate the main action in the service propositions
         # I guess it should be here
-        self.set_default_fields_and_objects_before_save()
+        acteur = self.set_default_fields_and_objects_before_save()
         self.full_clean()
-        return super().save(*args, **kwargs)
+        creating = self._state.adding  # Before calling save
```

Comment:
J'ajouterais bien un commentaire, car si on ne connaît pas les entrailles de Django, on serait amené à virer ça (ou ne pas le comprendre). 

```suggestion
        # This approach is inspired by the models.Model.save_base method
        # that sets this attribute only when an object is created.
        # 
        # This is similar to the created argument passed to post_save signal
        # https://github.com/django/django/blob/main/django/db/models/base.py#L954
        # https://docs.djangoproject.com/en/5.1/ref/signals/ 
        creating = self._state.adding  # Before calling save
```